plugins/hls-tactics-plugin/src/Wingman/Metaprogramming/Parser/Documentation.hs

```diff
@@ -6,8 +6,8 @@ import           Data.Functor ((<&>))
 import           Data.List (sortOn)
 import           Data.String (IsString)
 import           Data.Text (Text)
-import           Data.Text.Prettyprint.Doc hiding (parens)
-import           Data.Text.Prettyprint.Doc.Render.String (renderString)
+import           Prettyprinter hiding (parens)
```

Comment:
We tried this before! https://github.com/haskell/haskell-language-server/pull/2389

Reply:
Maybe we should just turn off deprecation warnings on these files locally?

Reply:
ouch, luckily you has noted it 😅 
maybe we can add cpp on prettyprinter version